app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<p>Details for a given users with post</p>
\ No newline at end of file
+<%= render 'shared/user', user: @user %>
+<p class="user-bio"><%= @user.bio %></p>
+
+<% if @user.posts.any? %>
+  <% @user.posts.each do |post| %>
```

Comment:
- [ ] The user's recent posts should be displayed on the user show page using the `recent_posts` method in your user model. Kindly do that.